src/Options.php

```diff
@@ -619,6 +619,28 @@ public function setMaxValueLength(int $maxValueLength): void
         $this->options = $this->resolver->resolve($options);
     }
 
+    /**
+     * Gets the http proxy setting.
+     *
+     * @return string
+     */
+    public function getHttpProxy(): ?string
+    {
+        return $this->options['http_proxy'];
+    }
+
+    /**
+     * Sets the http proxy. Be aware this option only works when curl client is used.
+     *
+     * @param string $httpProxy The http proxy
```

Comment:
I don't know why PHPStan doesn't report this, but the docblock is incorrect since the parameter is `nullable`